src/hls-adapter.js

```diff
@@ -671,6 +671,8 @@ export default class HlsAdapter extends BaseMediaSourceAdapter {
    */
   _parseVideoTracks(hlsVideoTracks: Array<Object>): Array<VideoTrack> {
     let videoTracks = [];
+    let audioTracks = [];
```

Comment:
Maybe something more descriptive like `audioOnlyTracks` that won’t be confused with regular audio tracks to the reader